### PR TITLE
Update Drabzit.json

### PR DESCRIPTION
**Pull request by Drabzit**
Dam Spaces

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4327"></a>

### CODE REVIEW

File: data/Drabzit.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-      "icon": "FaInstagram",
-      "Url": "https://www.instagram.com/drabz86/"
-    }
-  ]
-}
+    "name": "Chris Bardsley",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Living 1 Life at a Time, Crazy IT Dude, Small Fish in a big pond trying to get with the times and learn. Ready to Take next step in life and become more",
+    "avatar": "https://github.com/Drabzit.png",
```
Comment (Panquesito7) [inline data/Drabzit.json:6]: The avatar field is no longer required as of a new update.
```suggestion
```

File: data/Drabzit.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+    "tags": ["Open Source" , "GitHub", "Dad"],
+    "socials": [
+      {
+        "icon": "FaGithub",
+        "url": "https://github.com/Drabzit"
+      },
+      {
+        "icon": "FaLinkedin",
+        "url": "https:www.linkedin.com/in/christopher-bardsley-1291b0191"
```
Comment (Panquesito7) [inline data/Drabzit.json:15]: Wrong URL.
```suggestion
        "url": "https://www.linkedin.com/in/christopher-bardsley-1291b0191"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by arpitghura — APPROVED** [review]
Good

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**